Moya/Moya.swift

```diff
@@ -126,11 +129,12 @@ public class MoyaProvider<T: MoyaTarget> {
     public func request(token: T, completion: MoyaCompletion) -> Cancellable {
         let endpoint = self.endpoint(token)
         let request = endpointResolver(endpoint: endpoint)
+        let credential = credentialClosure?(token)
         let stubBehavior = self.stubBehavior(token)
 
         switch stubBehavior {
         case .NoStubbing:
-            return sendRequest(request, completion: completion)
+            return sendRequest(request, credential:credential, completion: completion)
         default:
             return stubRequest(request, completion: completion, endpoint: endpoint, stubBehavior: stubBehavior)
```

Comment:
Hmm, good question, the only reason I can think of is for debugging purposes maybe, to see if the credentials are set correctly for each request? Haven't really used the stubbing yet, so I'm not sure.